src/components/Product/Pipelines/index.js

```diff
@@ -538,7 +538,7 @@ function PipelinesPage({ location }) {
                     icon={<IconPlug />}
                     product={product.capitalized}
                     title="Ingest, transform, and send data between 25+ tools"
-                    description="Pre-built recipes make it easy to import data from a warehouse, sync with PostHog event data, and send to other products in your stack."
+                    description="PostHog's customer data platform makes it easy to import data from a warehouse, sync with event data, and send to other products in your stack."
```

Comment:
Oh no kidding, okay, let me dig around to see if there's any keyword data that can lead us here.

Reply:
Yeah, it's muddy because "CDP" is ALSO a non-profit, so it's unclear if the search volume being higher is because of that, or because that's what people call **customer data platform**. I'm good with belt-and-suspendering it like this.

<img width="549" alt="Screenshot 2024-11-05 at 12 56 24 PM" src="https://github.com/user-attachments/assets/72ffa485-34dd-43dc-a1f8-ccc6516790eb">

Reply:
Oh, one other backup: the slugs are consistently `cdp`, and having that in the path names will help with SEO as well.